### DISCUSSION TITLE
Google Sheet Only Showing Resolved Identities. Where Do I find Remaining Data

### DISCUSSION THREAD

**Discussion by PeterLangan** [merchantprotocol/theadslab#31] [Open Discussion Forum]
Hi All.

My Google Sheet is setup and syncing resolved identities of website visitors. However, I'm currently in the process of setting up retargeting and would like to locate the data I can use to retarget the people who PDOCs cannot resolve in full. Where can I locate this?

**Comment by jonathonbyrdziak** [comment]
I just rolled out a change yesterday, which puts all of the data on a single line for the webhook. I'll be transferring this over to the google sheet as well. So you'll have everything on one line.

Give me a couple days to get that done, because as soon as it is, you'll have exactly what you need.
